### PR TITLE
next events added

### PR DESCRIPTION
**Pull request by MeTaLPoU**

### CODE REVIEW

File: content/page/events.en.md [hunk 1]
```diff
@@ -8,6 +8,13 @@ draft: false
 Exodus Privacy is regulary performing talks about its works and results, in French or in English. If you want to invite us, feel free to [contact us](/en/page/who/).
 
 
+# Next events
+* [FR] [Festival des Libertés numériques](https://fdln.insa-rennes.fr/), INSA library, Rennes (France), 3th February 2020 
+* [FR] [Festival des Libertés numériques](https://fdln.insa-rennes.fr/), library of the University of Nantes (France), 11th February 2020
```
Comment (simpnu) [inline content/page/events.en.md]: I think the link (and maybe the name also) is not correct

File: content/page/events.en.md [hunk 2]
```diff
@@ -8,6 +8,13 @@ draft: false
 Exodus Privacy is regulary performing talks about its works and results, in French or in English. If you want to invite us, feel free to [contact us](/en/page/who/).
 
 
+# Next events
+* [FR] [Festival des Libertés numériques](https://fdln.insa-rennes.fr/), INSA library, Rennes (France), 3th February 2020
```
Comment (simpnu) [inline content/page/events.en.md]: `3rd` instead of `3th`

File: content/page/events.fr.md [hunk 3]
```diff
@@ -7,6 +7,14 @@ draft: false
 
 L'association Exodus Privacy propose régulièrement des conférences et interventions présentant son travail et ses résultats, en français ou en anglais. Si vous organisez un événement et souhaitez nous inviter, n'hésitez pas à [nous contacter](/fr/page/who/).
 
+
+# Prochains événements
+* [FR] [Festival des Libertés numériques](https://fdln.insa-rennes.fr/), Bibliothèque universitaire de l'INSA de Rennes, le lundi 3 février 2020 
+* [FR] [Festival des Libertés numériques](https://fdln.insa-rennes.fr/), Bibliothèque universitaire de Nantes, le mardi 11 février 2020
```
Comment (simpnu) [inline content/page/events.fr.md]: Same comment for the link and the name

File: content/page/events.en.md (hunk 1 above)
Comment (MeTaLPoU) [reply content/page/events.en.md]: The programmation of the 2020 edition is not published yet. So the link is the only one we have, and the name is ok.

**Pull request merged**